lib/windows/app/components/DeviceSetupDialog.jsx

```diff
@@ -106,8 +104,8 @@ export default class DeviceSetupDialog extends React.Component {
             <ConfirmationDialog
                 isVisible={isVisible}
                 isInProgress={isInProgress}
-                okButtonText={'Yes'}
-                cancelButtonText={'No'}
+                okButtonText="Yes"
```

Comment:
Curious, new rule in eslint to use double quote here?

Reply:
No, the rule says that curly-braces are not needed. Back to JSX syntax and double-quotes are required.